config.xhgui.yaml

```
@@ -10,14 +10,9 @@ hooks:
       fi
     service: db
   - exec: |
-      cat <<-EOF > .ddev/xhprof/xhprof_prepend.php
-      <?php
-      // DDEV's built in xhprof handler breaks our own.
-      // We'll temporarily override it here, but return control back later.
-      // If you don't want this behavior, comment out the hooks in ".ddev/config.xhgui.yaml".
-      return;
-      EOF
-  pre-stop:
+      cp .ddev/xhgui_prepend.php .ddev/xhprof/xhprof_prepend.php
+
+pre-stop:
```

The hook was not executed due to incorrect formatting:

```suggestion
  pre-stop:
```